test/lib/utils/addLiquidityHelper.ts

```diff
@@ -9,19 +9,19 @@ import {
     addLiquiditySingleTokenShouldHaveTokenInIndexError,
     AddLiquidityUnbalancedInput,
     Address,
-    BALANCER_ROUTER,
     NATIVE_ASSETS,
     PoolState,
     Slippage,
     Token,
     TokenAmount,
-    VAULT,
     Permit2Helper,
     AddLiquidityKind,
     ChainId,
     isSameAddress,
     PublicWalletClient,
 } from 'src';
+import { VAULT_V2 } from '@/utils/constantsV2';
+import { BALANCER_ROUTER } from '@/utils/constantsV3';
```

Comment:
I know this might be a bit annoying, but can you please export both constants files on `utils/index.ts` and import them from `src` here (and in other test files)?
This way, we make sure those constants are exposed externally.